src/main/io/vtx_rtc6705.c

```diff
@@ -69,6 +69,8 @@ bool vtxRTC6705Init(void)
 
     vtxCommonSetDevice(&vtxRTC6705);
 
+    vtxInit();
```

Comment:
This will break for targets defining `USE_VTX_RTC6705` but not `USE_VTX_COMMON`.

Reply:
Ah... it builds. However, `USE_VTX_COMMON` is required for VTX to be controllable by any means (including SIRINFPV style stick command only case). `io/vtx.c` itself is a heavy user of `vtxCommonXXX` and actually guarded with `#ifdef USE_VTX_COMMON`.

In summary, `USE_VTX_COMMON` is required for all VTX control capable builds.

Reply:
My original plan on `USE_VTX_COMMON` was to use it when something more than the basic VTX function the SIRIN-style stick command  and CLI variable is required, e.g., CMS.

In this regard, `io/vtx.c` should not depend on `vtxCommonXXX` functions. However, it was made to depend on the `vtxCommonXXX` some time in the past (it may have been not enough separation from the beginning of `vtxCommonXXX` introduction).

I would say we leave it as is for now, proceed with user defined table scheme and come back to review this problem when the transition is done.